server/main.py

```
@@ -35,7 +35,10 @@ def _interface_for(server_url: str) -> str | None:
     return None
 
 
-def main() -> None:
+def build_parser() -> argparse.ArgumentParser:
+    """הדגלים של השרת, בנפרד מההרצה — כדי שאפשר יהיה לפרוס שורת פקודה
+    בבדיקה בלי להרים uvicorn (‏#201: ה-e2e מרכיב argv, וצריך ראיה שהוא
+    מתפרס כאן ולא נבלע)."""
```

Ruff (0.16.2)

[error] Contains control characters that can permit obfuscated code

(PLE2502)